### DISCUSSION TITLE
What happens if the broadcast fails?

### DISCUSSION THREAD

**Discussion by ghost** [RGB-WG/rgb#196] [Q&A]
for example, the fee is too low, the psbt always boardcast failed, if there have any ways to RBF ?

**Comment by dr-orlovsky** [comment]
What do you mean by "psbt always boardcast failed"? Why do you broadcast PSBT? No need in doing that - and no standard way to do that.

If the fee for the witness tx is low, just bump it with RBF. RGB has nothing to do with that and is not affected.

**Reply by ghost** [reply]
1. after payer transfer and payee accept, how about the psbt， it no need to boardcast ?
2. if we bump it with RBF, i think the txid will change, the RGB20 witness tx also will changed, so the state should changed too, but there no ways to do that

**Reply by dr-orlovsky** [reply]
You do not broadcast PSBT, you broadcast signed and finalized transaction extracted from PSBT.

When the txid changes, the receiver seal does not (neither UTBXO blond seal nor witness-vased), as well terminal state transition doesn't change. So receiver can deduce that RBF had happened with no further messaging with sender.
